src/main/java/jdbchomework/entity/AbstractEntity.java

```diff
@@ -39,10 +50,16 @@ public String getName() {
     }
 
     public void setName(String name) {
+        if (name == null || name.equals("")) {
+            name = DEFAULT_NAME;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "name". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)